doc/api/modules.markdown

```diff
@@ -145,7 +145,7 @@ A module prefixed with `'./'` is relative to the file calling `require()`.
 That is, `circle.js` must be in the same directory as `foo.js` for
 `require('./circle')` to find it.
 
-Without a leading '/' or './' to indicate a file, the module is either a
+Without a leading '/', './', or '../' to indicate a file, the module is either a
 "core module" or is loaded from a `node_modules` folder.
```

Comment:
Unrelated-but-nearby nit: If you're feeling like it, `"core module"` in the next line should really just be `core module` without any quotation marks.